instrumentation/trace/context_test.go

```diff
@@ -45,3 +46,21 @@ func TestPropagateContextRetainsValue(t *testing.T) {
 	require.Equal(t, "foo", propgatedTracingContext.name)
 	require.NotEmpty(t, propgatedTracingContext.requestId)
 }
+
+func TestTranslateToRequestAndBack(t *testing.T) {
+	ctx := NewContext(context.Background(), "foo")
+	ep, _ := FromContext(ctx)
```

Comment:
What does the name `ep` stand for? not clear from code.

Reply:
it seems to be the convention in the actual context code so i used it.